validation/src/main/java/io/micronaut/validation/validator/DefaultValidator.java

```diff
@@ -645,14 +645,14 @@ private <T, E> Set<? extends ConstraintViolation<?>> validatePublisherValue(Argu
                                                                                 E value,
                                                                                 boolean canCascade
     ) {
-        DefaultConstraintValidatorContext<T> context = new DefaultConstraintValidatorContext<>(this, null, publisher, BeanValidationContext.fromGroups(groups));
-        try (ValidationPath.ContextualPath ignored = context.getCurrentPath().addReturnValueNode()) {
-            try (ValidationPath.ContextualPath ignored1 = context.getCurrentPath().addContainerElementNode("<publisher element>",
-                ValidationPath.DefaultContainerContext.ofIterableContainer(publisherArgument.getType()))) {
-                for (DefaultConstraintValidatorContext.ValidationGroup groupSequence : context.findGroupSequences()) {
-                    try (DefaultConstraintValidatorContext.GroupsValidation ignore = context.withGroupSequence(groupSequence)) {
-                        visitElement(context, publisher, valueArgument, value, canCascade);
-                    }
+        var context = new DefaultConstraintValidatorContext<>(this, null, publisher, BeanValidationContext.fromGroups(groups));
```

Comment:
Would you mind not making extra changes while fixing something, refactoring, etc., in another PR? Otherwise, it's hard to review what changed

Reply:
sure, reverted